WebDriverAgentLib/Routing/WebSocketScreenCasting.m

```diff
@@ -36,6 +36,7 @@ -(void) pushScreenShot:(SocketIOClient*) clientSocket andOrientation:(UIInterfac
 }
 
 -(void) pushRawScreenShot:(SocketIOClient*) clientSocket andOrientation:(UIInterfaceOrientation) orientation andScreenWidth:(CGFloat) screenWidth andScreenHeight:(CGFloat) screenHeight {
+  [NSThread sleepForTimeInterval:0.10f]; // decreases CPU usage
```

Comment:
Can you please help with what kind of commands you are using and how much time does it take ?
If this is only for particular use-case, can you please add comment on this and comment out this line so, whoever has the same use-case will enable it.

Reply:
A simple command of tap, takes between 1 and 2 seconds to interpret ... the interpretation of the command is much slower than the refresh rate of the screen ... 
I kept the sleep because it decreased the CPU usage in 40~50%  and did't decrease the refresh rate  so much compared to interpretation of the command.